Source/Scene/ModelExperimental/ModelExperimental.js

```diff
@@ -1233,32 +1233,11 @@ ModelExperimental.fromGltf = function (options) {
     ? ModelExperimentalType.TILE_GLTF
     : ModelExperimentalType.GLTF;
 
-  const modelOptions = {
-    loader: loader,
-    resource: loaderOptions.gltfResource,
-    type: type,
-    modelMatrix: options.modelMatrix,
-    scale: options.scale,
-    minimumPixelSize: options.minimumPixelSize,
-    maximumScale: options.maximumScale,
-    debugShowBoundingVolume: options.debugShowBoundingVolume,
-    cull: options.cull,
-    opaquePass: options.opaquePass,
-    allowPicking: options.allowPicking,
-    customShader: options.customShader,
-    content: options.content,
-    show: options.show,
-    color: options.color,
-    colorBlendAmount: options.colorBlendAmount,
-    colorBlendMode: options.colorBlendMode,
-    featureIdLabel: options.featureIdLabel,
-    instanceFeatureIdLabel: options.instanceFeatureIdLabel,
-    pointCloudShading: options.pointCloudShading,
-    lightColor: options.lightColor,
-    backFaceCulling: options.backFaceCulling,
-    shadows: options.shadows,
-    showCreditsOnScreen: options.showCreditsOnScreen,
-  };
+  const modelOptions = makeModelOptions(loader, type, options);
+  // Due to the logic above for determining the resources, use the computed
```

Comment:
Is there a reason for this comment? (i.e. is this something important that wasn't previously documented with the old model options object?)

Reply:
it was to point out that the variable name is slightly different (`resource` vs `gltfResource`). But eh that's clear enough from this line of code. I removed the comment.